### DISCUSSION TITLE
Kata 2- Ambiente virtual que se subirá al repo?

### DISCUSSION THREAD

**Discussion by JonathanGrimaldo** [LaunchX-InnovaccionVirtual/CursoIntroPython#197] [Q&A] [answered]
**Hola hice el ambiente virtual de la Kata 2, y genero muchos archivos en la carpeta lib, y es correcto.**

Mi pregunta es: 

**Si lo quiero subir a mi repo se omite la carpeta libs ó solo se suben capturas de pantalla para este ejercicio?**

_Anexo captura, ya que me genera +1k de cambios, de igual forma puedo omitirla.._

![image](https://user-images.githubusercontent.com/50802445/153722850-17996ccf-653f-4c7b-8917-9b5a08563ffc.png)

**Comment by ArmandoRiveraT** [comment, marked as answer]
Hola, saludos. Sube capturas de pantalla de las carpetas que se te generaron y una captura del cmd donde se vea el (env) activo

**Reply by JonathanGrimaldo** [reply]
De acuerdo, muchas gracias @ArmandoRiveraT

**Reply by DiegoPenalozaJuarez** [reply]
jejejeje porque hiciste eso? solo manda captura de pantalla :)
